.github/workflows/macos.yml

```diff
@@ -232,6 +232,7 @@ jobs:
             -Dbuildtype=release \
             -Ddefault_library=static \
             -Db_asneeded=true \
```

Comment:
Curious if `-Db_bitcode=true` can help the file-size even more than as-needed.
 - https://mesonbuild.com/Builtin-options.html#notes-about-apple-bitcode-support
 - https://help.apple.com/xcode/mac/11.0/index.html?localePath=en.lproj#/devde46df08a

Reply:
Bitcode is more for distribution via the App Store; it will on-demand recompile for e.g. new processor variants, or use app thinning to just give the slice relevant to a user's CPU.

Reply:
Ahh, thanks.  Didn't know if this is something Clang could do on our side.